src/js/components/form-elements/LabelField.jsx

```diff
@@ -71,7 +71,7 @@ const LabelField = (props) => {
               </a>
             </div>
             :
-            <div {...otherAttr} className={`font-size-xs ${className}`}>
+            <div style={{ color }} {...otherAttr} className={`font-size-xs ${className}`}>
```

Comment:
ok, sounds good. I was just aware of something similar to a common mistake when assigning a `className` dynamically like:
```javascript
 className={`${someBoolean && 'someClass'}`}
```
instead of:
```javascript
 className={`${someBoolean ? 'someClass' : ''}`}
```
for the first version if `someBoolean` would be `false`, we would assign `'false'` className.
And I was just wondering if something similar might happen when passing `null`/`undefined` to `style`, but I trust you.